app/src/main/kotlin/com/wire/android/ui/authentication/devices/register/RegisterDeviceViewModel.kt

```diff
@@ -80,6 +80,7 @@ class RegisterDeviceViewModel @Inject constructor(
                 continueEnabled = true,
                 error = RegisterDeviceError.InvalidCredentialsError
             )
+            RegisterClientResult.Failure.PasswordAuthRequired -> {}
```

Comment:
Wouldn't this be making the user stuck on the screen? As it doesn't change the state of `loading/continue button` or even displays an error message?

Reply:
Not sure but I think it will behave as before since this is not handled before

Reply:
we check if password is required is the `init` block before actually registering a client. But to make the logic more resilient we can log if   `PasswordAuthRequired` is returned at this step and update the screen state 

```kotlin
            RegisterClientResult.Failure.PasswordAuthRequired -> state = state.copy(
                loading = false,
                isPasswordRequired = true
            )
```